_articles/linux-terminal_for-beginners.md

```diff
@@ -26,25 +26,38 @@ A box like the one below will appear.
 
 When you run a command with **sudo** in front of it, You'll be prompted for your password, but when you type it, it won't show the letters as you type in the terminal, but type your password anyway and press <kbd>enter</kbd>.
 
-#### Frequently Used Commands
+#### Frequently Used Commands:
+*Omit the brackets when entering the command.*
+
+This series of commands will update the sources.list (a list telling your Ubuntu Operating System where to look for software and updates.)
+
+`sudo apt update`
+
+When using the 'upgrade' option of the command 'apt' your System will download and apply any updates to your System76.
+
+`sudo apt upgrade`
+
+The "man" command is short for manual. You can use it like so: 
+
+`man sources.list`
```

Comment:
Change this to something that is much more likely to be a first-timer's command.  Maybe `man sudo` or `man apt`?